linkerd/core/src/main/scala/io/buoyant/linkerd/Router.scala

```diff
@@ -113,6 +118,10 @@ trait RouterConfig {
   @JsonIgnore
   def label = _label.getOrElse(protocol.name)
 
+  /*
+   * interpreter controls how names are bound.
+   */
+
   @JsonProperty("interpreter")
```

Comment:
yeah, it's intended to basically be a section heading for the next few blocks. trying to provide visual cues about the sections